python/mxnet/contrib/quantization.py

```diff
@@ -459,7 +464,8 @@ def quantize_model(sym, arg_params, aux_params,
                    data_names=('data',), label_names=('softmax_label',),
                    ctx=cpu(), excluded_sym_names=None, excluded_op_names=None, calib_mode='entropy',
                    calib_data=None, num_calib_examples=None,
-                   quantized_dtype='int8', quantize_mode='smart', logger=None):
+                   quantized_dtype='int8', quantize_mode='smart',
+                   quantize_granularity='channel-wise', logger=None):
```

Comment:
Is the default value "channel-wise"?

Reply:
Good catch, forget to change back to `tensor-wise` as default value after validation :) Will change default value to `tensor-wise`.